rocksdb_replicator/rocksdb_replicator.cpp

```diff
@@ -101,22 +101,19 @@ ReturnCode RocksDBReplicator::addDB(const std::string& db_name,
   std::shared_ptr<DbWrapper> db_wrapper(
     new RocksDbWrapper(db_name, std::move(db))
   );
-  return addDB(db_name, db_wrapper, role, upstream_addr, replicated_db);
+  return addDB(db_name, db_wrapper, role, upstream_addr, replicated_db = replicated_db);
```

Comment:
Java take keyword arg? `replicated_db = replicated_db`?

Reply:
Don't worry this is in cpp :)

Reply:
o. sorry, I mean cpp. 
So, this is not keyboard arg as in python. You are assign value to itself, which is confusing.

Reply:
🤦‍♀️  too much switching languages recently... fixed